model_analyzer/reports/report_manager.py

```diff
@@ -613,7 +613,7 @@ def _create_summary_sentence(
     def _create_ensemble_summary_sentence(self, run_config: RunConfig) -> str:
         summary_sentence = "<BR><BR>"
         best_config_name = (
-            run_config.model_run_configs()[0].model_config().get_field("name")
+            run_config.model_run_configs()[0].model_config_variant().variant_name
```

Comment:
I think we can call `model_variant_name()` to avoid accessing `model_config_variant` inside ModelRunConfig.

Reply:
Done